ingenialink/poller.py

```diff
@@ -198,7 +198,7 @@ def _acquire_callback_poller_data(self):
         # Obtain current time
         t = delta.total_seconds()
 
-        self.__lock.acquire()
+        lock_poller.acquire()
```

Comment:
I'm not sure if we need to acquire the lock here. The lock will be acquired [here ](https://github.com/ingeniamc/ingenialink-python/pull/309/files#diff-828b231da05ac56f01ef705bb7514038b48b067402fba9c7d4fd4e409ac4577aL217) when reading a register.

Reply:
I think we can delete the lock here.